pytorch_lightning/utilities/imports.py

```diff
@@ -89,7 +89,7 @@ def _compare_version(package: str, op, version) -> bool:
 _TORCHVISION_AVAILABLE = _module_available("torchvision")
 _TORCHMETRICS_LOWER_THAN_0_3 = _compare_version("torchmetrics", operator.lt, "0.3.0")
 _TORCHMETRICS_GREATER_EQUAL_0_3 = _compare_version("torchmetrics", operator.ge, "0.3.0")
-_XLA_AVAILABLE = _module_available("torch_xla")
+_XLA_AVAILABLE: bool = _module_available("torch_xla")
```

Comment:
Why is this necessary?

`_module_available` is annotated with `-> bool`

Reply:
Unfortunately, mypy is not able to figure that out without this explicit type hint.

Reply:
can confirm, we had the same problem in metrics:
https://github.com/PyTorchLightning/metrics/blob/525642df8c3f3f5f93cdd22b6b9cd802a3854634/torchmetrics/utilities/imports.py#L79-L80